markets/perps-market/contracts/interfaces/IAsyncOrderModule.sol

```diff
@@ -49,12 +91,51 @@ interface IAsyncOrderModule {
         bytes extraData
     );
 
+    /**
+     * @notice Commit an async order via this function
+     * @param commitment Order commitment data (see AsyncOrder.OrderCommitmentRequest struct).
+     * @return retOrder order details (see AsyncOrder.Data struct).
+     * @return fees order fees (protocol + settler)
+     */
     function commitOrder(
         AsyncOrder.OrderCommitmentRequest memory commitment
     ) external returns (AsyncOrder.Data memory retOrder, uint fees);
 
+    /**
+     * @notice Cancel an expired order via this function
+     * @param marketId Id of the market used for the trade.
+     * @param accountId Id of the account used for the trade.
+     */
     function cancelOrder(uint128 marketId, uint128 accountId) external;
 
+    /**
+     * @notice Settles an offchain order. It's expected to revert with the OffchainLookup error with the data needed to perform the offchain lookup.
+     * @param marketId Id of the market used for the trade.
+     * @param accountId Id of the account used for the trade.
+     */
+    function settle(uint128 marketId, uint128 accountId) external view;
```

Comment:
I've added these too in the new AsyncOrderSettlementModule (but currently without comment). No biggie but might have to rebase